WORKSPACE

```diff
@@ -432,6 +432,14 @@ rpm(
     ],
 )
 
+rpm(
+    name = "acl-0__2.3.1-3.el9.s390x",
+    sha256 = "1ea7f709dde5cc1bb78da69989524b6dadb142f5ff71d5c19ff9e9b0559d6633",
+    urls = [
+        "http://mirror.stream.centos.org/9-stream/BaseOS/s390x/os/Packages/acl-2.3.1-3.el9.s390x.rpm",
+    ],
+)
+
```

Comment:
Obviously you're going to need to refresh this as you rebase on top of `main`. When you do, please have a commit at the beginning of the series that updates RPMs for existing architectures, so that when you add s390x into the mix we will see matching package versions.

Reply:
I can do that, if you want it in a separate commit. Though my plan would have been to just run `make rpm-deps` for all arches at once.